esp-hal/src/interrupt/xtensa.rs

```diff
@@ -1,3 +1,5 @@
+//! Interrupt handling - Xtensa
```

Comment:
```suggestion
//! Interrupt handling
```

I think we probably don't need to specify the arch here. We build the documentation for each chip, so each corresponding set of documentation should only include the relevant bits anyway.